src/testdir/test_lua.vim

```diff
@@ -20,3 +20,20 @@ func Test_luado()
   bwipe!
   bwipe!
 endfunc
+
+func Test_set_cursor()
+  " Check that setting the cursor position works.
+  new
+  call setline(1, ['first line', 'second line'])
+  normal gg
+  lua << EOF
+w = vim.window()
+w.line = 1
+w.col = 5
+EOF
+  call assert_equal([1, 5], [line('.'), col('.')])
```

Comment:
you can also use `getcurpos()` to get the curswant column.

Reply:
Would you like me to change these?  

I'm content that the test as written captures what I care about - but also more than happy to be guided in the right direction...

Reply:
That was merely a hint, that you can use `getcurpos()` function, which returns all needed infos (column, line, and also curswant property). you don't have to change it, the current test would work as well, I guess.